routes/api_v1.php

```diff
@@ -1489,6 +1490,17 @@
                 Route::group(array('prefix' => 'allowed-extra-questions'), function () {
                     Route::get('', [ 'uses' => 'OAuth2SummitOrderExtraQuestionTypeApiController@getAttendeeExtraQuestions']);
                 });
+
+                // attendee notes
+                Route::group(array('prefix' => 'notes'), function () {
+                    Route::get('', 'OAuth2SummitAttendeeNotesApiController@getAttendeeNotes');
+                    Route::post('', ['middleware' => 'auth.user', 'uses' => 'OAuth2SummitAttendeeNotesApiController@addAttendeeNote']);
+                    Route::group(array('prefix' => '{note_id}'), function () {
+                        Route::get('', 'OAuth2SummitAttendeeNotesApiController@getAttendeeNote')->where('note_id', '[0-9]+');
```

Comment:
you could define note_id where at group level
 Route::group(['prefix' => '{note_id}', 'where' => '']),

Reply:
done